### DISCUSSION TITLE
No local commits and changes but github said there are commits to be pushed

### DISCUSSION THREAD

**Discussion by Juanryhn** [community/community#23078] [Projects and Issues]
<p>currently I was using github desktop app, but I don’t understand what this means ?. Cause there are no local changes and commits in my repo but I still got this message from github<br>
<img src="https://user-images.githubusercontent.com/87475729/181074714-e82fd8f8-9ff1-48a2-a8f9-9da9ad03ebc0.png" alt="image" data-base62-sha1="kDJlj30U6isBtJ8wlD7CL1bmBer" width="632" height="236"><br>
and same with the vs code. Can anyone explain to me how to fix this and what should I do to avoid this issue ?<br>
<img src="https://user-images.githubusercontent.com/87475729/181074715-e3833b04-f760-4ef8-a339-0cc9d83aa4b7.png" alt="image" data-base62-sha1="y1Jt5HWz1kCpU705XT1cwcSf9ey" width="167" height="55"></p>

**Comment by jfullstackdev** [comment, marked as answer]
<p>after you commit, you should push that to origin by clicking it, it will upload all those local files to the repo on GitHub</p>
